test/paymaster/singleton/verifying-singleton-paymaster.ts

```diff
@@ -117,7 +117,11 @@ describe("EntryPoint with VerifyingPaymaster", function () {
       entryPoint
     );
 
-    const hash = await verifyingSingletonPaymaster.getHash(userOp1);
+    const hash = await verifyingSingletonPaymaster.getHash(
+      userOp1,
+      0,
```

Comment:
I would suggest actually getting nonce from the contract here

same on line 160-164

Reply:
done

Reply:
if looks good do approve